Tests/xcodeprojMeasurementsTests/MeasurementsTests.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+import PathKit
+import SwiftShell
+@testable import xcodeproj
+import XCTest
+
+final class MeasurementsTests: XCTestCase {
+    
+    let tempDirectory = Path.temporary
+    var projectPath: Path?
```

Comment:
We can use explicitly unwrapped optionals here. It's safe to use them as long as we initialize them in the `setUp` method.